core/controllers/base.py

```diff
@@ -53,7 +53,11 @@
 
 
 def _clear_login_cookies(response_headers):
-    """Clears login cookies from the given response headers."""
+    """Clears login cookies from the given response headers.
+
+    Args:
+        response_headers: str. response headers.
```

Comment:
Please write complete clauses that start with a capital letter and end with a period. E.g.: "A set of HTTP response headers."

Also, how do you know it's a str? I don't think strs usually have an add_header() method. Please use type() to check what the types are, and represent them correctly in the docstring -- it's not much use having docstrings if they're incorrect.

Note that the above comments apply to all docstrings in this PR.